console/program/src/request/serialize.rs

```diff
@@ -51,6 +52,8 @@ impl<'de, N: Network> Deserialize<'de> for Request<N> {
                 Ok(Self::from((
                     // Retrieve the caller.
                     serde_json::from_value(request["caller"].clone()).map_err(de::Error::custom)?,
+                    // Retrieve the parent.
+                    serde_json::from_value(request["parent"].clone()).map_err(de::Error::custom)?,
```

Comment:
```suggestion
                    serde_json::from_value(request["parent"].take()).map_err(de::Error::custom)?,
```
(this change will work without further adjustments after a rebase)